faces-tck/pom.xml

```diff
@@ -0,0 +1,162 @@
+                                </artifactItem>
+                            </artifactItems>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>replace-glassfish-hardcoded-exec</id>
+                        <phase>pre-integration-test</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <target>
+                                <echo>Replacing Glassfish hardcoded control with Payara</echo>
+                                <copy file="pom.xml-old-tck-run"
+                                    tofile="${project.build.directory}/faces-tck-${faces.version}/tck/old-tck/run/pom.xml" 
+                                    overwrite="true" />
+                            </target>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <phase>pre-integration-test</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <executable>${maven.executable}</executable>
+                    <commandlineArgs>${maven.exec.cmdline}</commandlineArgs>
+                    <environmentVariables>
+                        <LC_ALL>C</LC_ALL>
+                    </environmentVariables>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+    
+    <profiles>
+        
+        <profile>
+            <!-- Skip old tests with '-DskipOld'.-->
+            <id>skipOld</id>
+            <activation>
+              <property>
+                <name>skipOld</name>
+              </property>
+            </activation>
+            <properties>
+                <maven.exec.projects>-pl -:old-faces-tck-parent,-:old-tck-build,-:old-tck-run -Dtck.old.skip=true</maven.exec.projects>
+            </properties>
+        </profile>
+        
+        
+        <profile>
+            <!-- Skip new tests with '-DskipNew'.-->
+            <id>skipNew</id>
+            <activation>
+              <property>
+                <name>skipNew</name>
+              </property>
+            </activation>
+            <properties>
+                <maven.exec.projects>-pl :old-faces-tck-parent,:old-tck-build,:old-tck-run</maven.exec.projects>
+            </properties>
+        </profile>
```

Comment:
If we've only got a runner for payara-ci-remote, it may be worth adding a dummy profile for server-managed to skip the tests (or just adding a README which says "no support for managed profile yet".